### PR TITLE
Mtm49309 add compatibility policy to concepts guide

### PR DESCRIPTION
**Pull request by dianaolives**

### CODE REVIEW

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+weight: 75
+---
+
+{{< product-c8y-iot >}} provides the highest possible level of compatibility to make sure that your investments into developing solutions with {{< product-c8y-iot >}} are preserved. The current compatibility statements are as follows:
+
+#### API compatibility
+
+{{< product-c8y-iot >}}'s REST, SmartREST and MQTT APIs as documented in the product documentation are backwards compatible.  You can find such documentation at:
+
+* [OpenAPI Specifications - {{< product-c8y-iot >}}](https://cumulocity.com/api/)
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/compatibility-policy.md:12]: OpenAPI Specifications - {{< product-c8y-iot >}} > we have a particular variable for "Cumulocity IoT OpenAPI Specification" which is {{< openapi >}} and which we can use here

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* [SmartREST 2.0](/reference/smartrest-two/)
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/compatibility-policy.md:13]: add "in the Reference guide"

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* [Device integration using MQTT](/device-integration/mqtt/)
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/compatibility-policy.md:14]: add "in the Reference guide"

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+
+However, {{< product-c8y-iot >}} is continually improving the user experience and product capabilities and may improve the API from time-to-time. Therefore, applications shall always obey a few basic rules:
+
+* Clients shall only use documented API methods and documented behavior. Do not rely on undocumented but observed behavior.
+* Clients shall only rely on behavior that is explicitly described in the documentation. For instance, clients shall not rely on a sequence of results if no sort order is guaranteed.
+* Clients can rely on a stable inventory API where they can manage their inventory objects. However, the data structures for objects owned by the {{< product-c8y-iot >}} platform can change.
+* APIs are free to return more JSON fields and clients must be written in a way that they ignore such fields.
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/compatibility-policy.md:22]: perhaps add a comma after "fields" to improve readability?

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+{{< /c8y-admon-info >}}
+
+#### Beta APIs
+
+To maintain API and product quality we may, from time to time, hold back new APIs in a beta state to make sure that they are robust, properly documented and provide the capability required. These APIs may change without a prior announcement. Please keep this in mind if you start using APIs that are documented as beta on our [API documentation](https://cumulocity.com/api/).
+
+#### SDK and client library compatibility
+
+{{< product-c8y-iot >}} Core libraries and software development kit APIs (Java/JavaScript/C++/…) may be changed. The libraries and SDKs help developers to access the REST and MQTT APIs in their custom implementation and are typically bundled with the custom implementation. In case {{< product-c8y-iot >}} provides a new version of the libraries, it is not required to upgrade the custom implementation since the underlying REST and MQTT APIs remain compatible as long as no breaking changes happen (see [API compatibility](/concepts/introduction/#api-compatibility)).
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/compatibility-policy.md:35]: Core > core
software development kit > SDK 
(Java/JavaScript/C++/…) > (like Java, JavaScript, C++)
(/concepts/introduction/#api-compatibility)) > The link points to a section in the same file so it is sufficient to have (#api-compatibility) as link. But you need to add an anchor above this heading to make it work :-)

File: content/concepts/introduction-bundle/compatibility-policy.md (hunk 2 above)
Comment (dianaolives) [reply content/concepts/introduction-bundle/compatibility-policy.md:13]: Can it be "Reference guide > SmartREST 2.0"?

File: content/concepts/introduction-bundle/compatibility-policy.md (hunk 3 above)
Comment (dianaolives) [reply content/concepts/introduction-bundle/compatibility-policy.md:14]: Device integration using MWTT isn't in the reference guide is it? In the documentation it shows its under "Device integration.

Comment (BeateRixen) [reply content/concepts/introduction-bundle/compatibility-policy.md:14]: Yes, sorry, you are right. It is in the Device SDK guide.

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+{{< /c8y-admon-info >}}
+
+#### Beta APIs
+
+To maintain API and product quality we may, from time to time, hold back new APIs in a beta state to make sure that they are robust, properly documented and provide the capability required. These APIs may change without a prior announcement. Please keep this in mind if you start using APIs that are documented as beta on our [API documentation](https://cumulocity.com/api/).
+
+#### SDK and client library compatibility
+
+{{< product-c8y-iot >}} core libraries and software development kit APIs (like Java, JavaScript, C++) may be changed. The libraries and SDKs help developers to access the REST and MQTT APIs in their custom implementation and are typically bundled with the custom implementation. In case {{< product-c8y-iot >}} provides a new version of the libraries, it is not required to upgrade the custom implementation since the underlying REST and MQTT APIs remain compatible as long as no breaking changes happen (see [API compatibility](#api-compatibility)).
```
Comment (BeateRixen) [inline content/concepts/introduction-bundle/compatibility-policy.md:36]: software development kit > SDK
Did you leave this intentionally? I would actually change it to SDK as we use the acronym everywhere, also in the heading, and in this context we do not need to explain what the acronym stands for

Comment (dianaolives) [reply content/concepts/introduction-bundle/compatibility-policy.md:36]: I copied it exactly as the word document. I hadn't noticed that detail. I'll change it :)

**Review by BeateRixen — APPROVED** [review]

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+title: Compatibility policy
+weight: 75
+---
+
+{{< product-c8y-iot >}} provides the highest possible level of compatibility to make sure that your investments into developing solutions with {{< product-c8y-iot >}} are maintained. The current compatibility statements are described as follows.
+
+<a name="api-compatibility"></a>
+#### API compatibility
+
+{{< product-c8y-iot >}}'s REST, SmartREST and MQTT APIs as documented in the product documentation are backwards compatible.  You can find such documentation in:
```
Comment (MWindel) [inline content/concepts/introduction-bundle/compatibility-policy.md:11]:
```suggestion
{{< product-c8y-iot >}}'s REST, SmartREST and MQTT APIs as documented in the product documentation are backwards compatible. You can find such documentation in:
```

File: content/concepts/introduction-bundle/compatibility-policy.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+* APIs are free to return more JSON fields, therefore, clients must be written in a way that they ignore such fields.
+
+
+{{< c8y-admon-info >}}
+If changes result in breaking backward compatibility, to provide our customers with the time to change their solution, such changes will be announced at least one GA release ahead of the GA release where the change becomes effective. Where the change has a wide-ranging impact, we will delay taking action until we have ascertained that the impact on our customers is minimal.
+{{< /c8y-admon-info >}}
+
+#### Beta APIs
+
+To maintain API and product quality we may, from time to time, hold back new APIs in a beta state to make sure that they are robust, properly documented and provide the capability required. These APIs may change without a prior announcement. Please keep this in mind if you start using APIs that are documented as beta on our [API documentation](https://cumulocity.com/api/).
```
Comment (MWindel) [inline content/concepts/introduction-bundle/compatibility-policy.md:32]:
```suggestion
To maintain API and product quality we may, from time to time, hold back new APIs in a beta state to make sure that they are robust, properly documented, and provide the capability required. These APIs may change without a prior announcement. Please keep this in mind if you start using APIs that are documented as beta on our [API documentation](https://cumulocity.com/api/).
```

File: content/concepts/introduction-bundle/compatibility-policy.md (hunk 7 above)
Comment (dianaolives) [reply content/concepts/introduction-bundle/compatibility-policy.md:11]: Done

File: content/concepts/introduction-bundle/compatibility-policy.md (hunk 8 above)
Comment (dianaolives) [reply content/concepts/introduction-bundle/compatibility-policy.md:32]: Done

**Review by MWindel — APPROVED** [review]

**Pull request merged**